tests/integration/Mailer/WordPress/WordpressMailerTest.php

```diff
@@ -16,6 +20,9 @@ class WordpressMailerTest extends \MailPoetTest {
   public function _before() {
     $this->subscribersRepository = $this->diContainer->get(SubscribersRepository::class);
     $this->subscribersRepository->truncate();
+
+    $settings = SettingsController::getInstance();
```

Comment:
This is just temporary and is used in a test that needs to be refactored (or more likely moved somewhere else).